src/AWS.Deploy.CLI/ServerMode/Controllers/DeploymentController.cs

```diff
@@ -368,6 +368,13 @@ public IActionResult StartDeployment(string sessionId)
             if (state.SelectedRecommendation == null)
                 throw new SelectedRecommendationIsNullException("The selected recommendation is null or invalid.");
 
+            var systemCapabilityEvaluator = serviceProvider.GetRequiredService<ISystemCapabilityEvaluator>();
+
+            var capabilities = await systemCapabilityEvaluator.EvaluateSystemCapabilities(state.SelectedRecommendation);
+
+            if (capabilities.Any())
+                return Problem("Unable to start deployment due to missing system capabilities.");
```

Comment:
We should return back what are the missing capabilities in the error message.

Reply:
I agree with @normj assuming this message will be shown in some window/label/alert and we would like to be very specific here so that customer can take appropriate action.

Reply:
Updated